src/Deferred.php

```diff
@@ -14,6 +14,9 @@ public function __construct(callable $canceller = null)
         $this->canceller = $canceller;
     }
 
+    /**
+     * @inheritdoc
+     */
```

Comment:
These look pretty much useless and I suppose if your IDE really depends on these, you may want to switch to another one :-)